ui/resolution_selection.js

```diff
@@ -179,6 +179,132 @@ shaka.ui.ResolutionSelection = class extends shaka.ui.SettingsMenu {
   }
 
 
+  /** @private */
+  updateResolutionSelectionnew_() {
```

Comment:
When my goal as a reviewer is to determine what behavior changed, you make it extremely difficult when you duplicate 100+ lines of code.  Please do not do this.

Duplicate code is also more difficult to debug and maintain.

Reply:
I had added new function because to I can't make the changes in old function because in order to perform the changes in old function I had to pass argument or declare global variable.

Reply:
Can you tell me how I can declare a global variable that I can use in line 50 this.event manager and updateresolutionscheme? I tried but I can't declare. global variable.

Reply:
updateResolutionSelection_ and updateResolutionSelectionnew_ have the same code I have added code from line 211 to 225